keyboards/keycapsss/kimiko/readme.md

```diff
@@ -0,0 +1,23 @@
+# Kimiko
+
+<img src="https://keycapsss.com/media/image/7f/88/2b/kimiko-split-keyboard-1.jpg" width="800" />
+
+A split keyboard with 4x6 vertically staggered keys and thumb keys.
+
+- Per key RGB led's (SK6812 Mini-E)
+- 6 underglow RGB led's per side (SK6812 Mini)
+- Support for 1 rotary encoder per side (two possible positions)
+
+Keyboard Maintainer: [BenRoe](https://github.com/BenRoe/) [@keycapsss](https://twitter.com/keycapsss)  
+Hardware Supported: Pro Micro 5V/16Mhz and compatible  
+Hardware Availability: [keycapsss.com](https://keycapsss.com)
+
+Make firmware .hex for this keyboard (after setting up your build environment):
+
+    make keycapsss/kimiko:default
+
+Example of flashing this keyboard (or use [QMK Toolbox](https://github.com/qmk/qmk_toolbox)):
+
+    make keycapsss/kimiko:default:dfu
```

Comment:
```suggestion
    make keycapsss/kimiko:default:flash
```

Reply:
@zvecr qmk choose the right programmer depending on the `rules.mk`?

Reply:
Yes. And specifically, based on the bootloader set, and MCU type.